File: app/src/main/java/com/example/questionarioimpacto/SplashActivity.java
package com.example.questionarioimpacto;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_welcome);
        Handler handle = new Handler();
        handle.postDelayed(new Runnable() {
            @Override public void run() {
                mostrarLogin();
            }
        }, 2000);
    }

    private void mostrarLogin() {
        Intent intent = new Intent(SplashActivity.this, TermsConditionsActivity.class);
        startActivity(intent);
        finish();
    }
}

File: app/src/main/java/com/example/questionarioimpacto/SplashActivity.kt
package com.example.questionarioimpacto

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_welcome)
        val handle = Handler()
        handle.postDelayed({ mostrarLogin() }, 2000)

    }

    private fun mostrarLogin() {
        val intent = Intent(this@SplashActivity, TermsConditionsActivity::class.java)
        startActivity(intent)
        finish()
    }
}